packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -34,11 +38,15 @@ const trace = makeTracer('VM', false);
  *  compoundedInterest: Ratio,
  *  interestRate: Ratio,
  *  latestInterestUpdate: bigint,
- *  totalDebt: Amount<'nat'>,
```

Comment:
This caused a null reference error in the UI https://github.com/Agoric/dapp-treasury/blob/main/ui/src/components/vault/VaultConfigure/VaultConfigure.jsx#L200. I guess typescript/CI didn't catch this somehow...

Reply:
Ah, I didn't realize that was being used. Yeah, the dapp-treasury repo currently has no type checking https://github.com/Agoric/dapp-treasury/issues/97